baseband/formats.py

```diff
@@ -0,0 +1,48 @@
+# Licensed under the GPLv3 - see LICENSE
+"""Routines to obtain information on baseband files."""
+import importlib
+
+
+FILE_FORMATS = {'dada': 'DADA',
+                'mark4': 'Mark4',
+                'mark5b': 'Mark5B',
+                'vdif': 'VDIF',
+                'gsb': 'GSB'}
+
+
+def file_info(name, fmt=None, **kwargs):
```

Comment:
Not a huge fan of `fmt`; would prefer `format` (can still use `fmt` in the for loop).